docs/FAQ.md

```diff
@@ -0,0 +1,83 @@
+# Registry past-and-present cheat sheet
+
+> Everything You Always Wanted to Know about the Registry
+
+## What's a Registry?
+
+It's a server-side storage and delivery software that let you publish and exchange docker images.
+
+Whenever you do a `docker pull` or `docker push`, your engine communicates with a registry.
+
+## So, does the Docker Hub contain a registry?
+
+Yes.
+
+That "Hub registry" is operated by Docker.
+
+It is free to use for public content, and offers paid-for private storage as well.
+
+Support for the hub is handled directly by Docker, over here: https://support.docker.com
+
```

Comment:
I don't want to overload registry documentation with details about other products (like the Hub). If people want to learn about the hub they should go there (furthermore, if they landed here, they are already pretty deep in).

One of the defect of Docker documentation at large IMHO is how cluttered and unwieldy it is.